app/src/main/java/com/globant/scriptsapadea/ui/fragments/PictureFragment.java

```diff
@@ -0,0 +1,96 @@
+package com.globant.scriptsapadea.ui.fragments;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.globant.scriptsapadea.R;
+import com.globant.scriptsapadea.manager.ActivityResultEvent;
+import com.squareup.otto.Subscribe;
+
+/**
+ * Created by leonel.mendez on 5/19/2015.
+ */
+public class PictureFragment extends BaseFragment implements View.OnClickListener{
+
+
+    private static int GALLERY = 0x001;
+    private static int CAMERA = 0x010;
+
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+
+
```

Comment:
Empty spaces & tabs all over the code. Use Android Studio code formatter to avoid this.